README.md

```diff
@@ -27,35 +27,35 @@ PRs should be posted _before_ class begins on Monday, March 4, 2024.
 
 ### React Challenge
 
-1. React Debugging: The Card Draw React application has three (3) errors that need to be fixed. Use the console and error messages to guide you to the solution.
+1. React Debugging: The Card Draw React application has three (3) errors that need to be fixed. Use the console and error messages to guide you to the solution.✅
 
-2. React Feature: Add functionality to the `Return Cards and Shuffle Deck` button using the `shuffle` function to reset the Card Draw application to its original state.
+2. React Feature: Add functionality to the `Return Cards and Shuffle Deck` button using the `shuffle` function to reset the Card Draw application to its original state.✅
 
-3. React Concepts: Add comments to the seven (7) notations in the `drawCard` function describing in detail the functionality line of code BELOW the comment.
+3. React Concepts: Add comments to the seven (7) notations in the `drawCard` function describing in detail the functionality line of code BELOW the comment.✅
 
-4. STRETCH (optional): Add stylings of your choice to the application. Ensure the styling does not interfere with the card draw functionality.
+4. STRETCH (optional): Add stylings of your choice to the application. Ensure the styling does not interfere with the card draw functionality.✅
 
 ### Interview Practice Questions
 
 1. What are three (3) syntax differences between HTML and JSX?
 
--
--
--
+- The casing is different. Pascal casing with class is different.
+- I also feel that HTML is more relaxed then JSX. JSX stops you in your track with an error and you can't see the screen.
+- JSX has to have curly braces. You need curly braces to put your JavaScript in JSX.
 
-2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
+2. What is yarn? What file(s) are modified when you run the command yarn in your terminal? Yarn is a package manager. Yarn modifies the package.json. 
```

Comment:
there are a few other things to consider.  You're partly correct! Double check what happens when your run yarn after pulling a React app. :)